packages/js/components/src/woo-product-tab-item/woo-product-tab-item.tsx

```diff
@@ -10,16 +10,20 @@ import { createElement, Fragment } from '@wordpress/element';
  */
 import { createOrderedChildren } from '../utils';
 
+export type ProductFillLocationType< T = Record< string, unknown > > = {
+	name: string;
+	order?: number;
+	fillProps?: T;
```

Comment:
Note the addition of `fillProps` on this type (used in all product slots) with a generic type.

Reply:
After looking at the usage of this, I think it may be nicer to have `fillProps` as a callback function on the SlotFill wrapper that provides the `name` as the first parameter.
That way we won't need to nest to deep and allows us to keep the locations array pretty clean and simple.